dashboard/src/containers/Root.tsx

```diff
@@ -30,6 +31,8 @@ class Root extends React.Component {
   } = {
     "/": AppList,
     "/apps/:namespace/:releaseName": AppView,
+    "/apps/edit/:namespace/:releaseName": AppEdit,
+    "/apps/edit/:namespace/:releaseName/:version": AppEdit,
```

Comment:
additional route to specify a new version for a release

Reply:
we can drop this now right?